android/WALT/app/src/main/java/org/chromium/latency/walt/ClockManager.java

```diff
@@ -478,15 +480,22 @@ TriggerMessage parseTriggerMessage(String s) {
             handler = new Handler();
         }
 
-        private void go(final TriggerMessage tmsg) {
+        private void go(final String s) {
             handler.post(new Runnable() {
                 @Override
                 public void run() {
-                    onReceive(tmsg);
+                    onReceiveRaw(s);
                 }
             });
         }
 
+        void onReceiveRaw(String s) {
+            if (TriggerMessage.isTriggerString(s)) {
+                TriggerMessage tmsg = new TriggerMessage(s.substring(1).trim());
+                onReceive(tmsg);
+            }
```

Comment:
Done as Log.i(), there is no easy access to on-screen logger here.
I think currently we do get non trigger strings here when the listener is running and we send a command that replies with and ack.